docs/validation/guides/validate-ODF-LSO.md

```diff
@@ -0,0 +1,44 @@
+# Validate Deployment of OpenShift Data Foundation (ODF) with LSO
+
+The steps below describe how to run acceptance tests manually for ODF (OpenShift Data Foundation) using LSO (Local Storage Operator).
+
+Prerequisites
+
+- OpenShift Cluster Installed in version 4.13 (or newer). Export the variable `CLUSTER_VERSION`. Example: `export CLUSTER_VERSION=4.13`
+- Cluster name exported in the variable `CLUSTER_NAME`
+- Cluster administrator credentials is saved under ${INSTALL_DIR}/auth, example:
+
+~~~bash
+$ tree ${INSTALL_DIR}
+└── auth
+    ├── kubeadmin-password
+    └── kubeconfig
+~~~
+
+Steps:
+
+1. Run a simple test:
+
+~~~bash
+podman run -v ${INSTALL_DIR}:/opt/cluster:Z quay.io/ocsci/ocs-ci-container:stable run-ci --cluster-path /opt/cluster --ocp-version ${CLUSTER_VERSION} --ocs-version ${CLUSTER_VERSION} --cluster-name ${CLUSTER_NAME} tests/manage/z_cluster/test_noobaa_xss_vulnerability.py --dev-mode
+~~~
+
+3. Run acceptance suite:
+
+~~~bash
+podman run -v ${INSTALL_DIR}:/opt/cluster:Z quay.io/ocsci/ocs-ci-container:stable run-ci -m acceptance --junit-xml /opt/cluster/junit.xml --cluster-path /opt/cluster --ocp-version ${CLUSTER_VERSION} --ocs-version ${CLUSTER_VERSION} --color=yes --cluster-name ${CLUSTER_NAME} tests/
+~~~
+
+The results must be saved in `${INSTALL_DIR}/junit.xml`.
+
+3. Collect the Must Gather for ODF
+
+~~~bash
+oc adm must-gather --image=registry.redhat.io/odf4/ocs-must-gather-rhel8:v4.9
```

Comment:
the image `registry.redhat.io/odf4/ocs-must-gather-rhel8:v4.9` is unavailable.

https://docs.openshift.com/container-platform/4.10/support/gathering-cluster-data.html

Reply:
Can you try this image `registry.redhat.io/odf4/odf-must-gather-rhel9`?
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/default_config.yaml#L125